src/app/meshes/views/MeshView.vue

```diff
@@ -1,30 +1,7 @@
 <template>
-  <RouteView
-    v-slot="{
-      children
-    }"
-  >
+  <RouteView>
     <AppView>
-      <KTabs
-        class="route-mesh-view-tabs"
-        :tabs="items"
-        :has-panels="false"
-        :model-value="(items.find(item => children.includes(data.get(item.hash)?.module as string)) ?? items[0]).hash"
-      >
-        <template
-          v-for="item in items"
-          :key="`${item.hash}-anchor`"
-          #[`${item.hash.substr(1)}-anchor`]
-        >
-          <router-link
-            :to="{
-              name: item.hash.substr(1),
-            }"
-          >
-            {{ item.title }}
-          </router-link>
-        </template>
-      </KTabs>
+      <NavTabs :tabs="tabs" />
```

Comment:
Sorry just looking at this again following the last lot of changes plus our GUI sync.

Am I right in thinking that in the case of Mesh Tabs `NavTabs` is only replacing:

`(items.find(item => children.includes(data.get(item.hash)?.module as string)) ?? items[0]).hash`

?

Now I can see that we have left more of the "50% towards doing this automatically" in here I'm not as concerned.

Reply:
Yes, in essence, the changes to MeshView are relatively small now. Some changes in generating the tabs array were necessary as a result but the approach is almost the same